<div id="dlpage">
    <h1>Quantitative Biology </h1>
    <h2>New submissions</h2>
    <div class="list-dateline">Submissions received from Fri 3 May 24 to Mon 6 May 24, announced Tue, 7 May 24</div>
    <ul>
        <li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
        <li><a href="#item16">Cross-lists</a></li>
        <li><a href="#item21">Replacements</a></li>
    </ul>
    <small>[ total of 31 entries: <b>1-31</b> ]</small><br>
    <small>[ showing up to 2000 entries per page: <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> | <font
            color="#999999">more</font> ]</small><br>
    <h3>New submissions for Tue, 7 May 24</h3>
    <dl>
        <dt><a name="item1">[1]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02374"
                    title="Abstract">arXiv:2405.02374</a> [<a href="/pdf/2405.02374" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02374" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Protein binding affinity prediction under multiple
                    substitutions applying eGNNs on Residue and Atomic graphs combined with Language model information:
                    eGRAL
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Fiorellini-Bernardis%2C+A">Arturo
                        Fiorellini-Bernardis</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Boyer%2C+S">Sebastien Boyer</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Brunken%2C+C">Christoph Brunken</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Diallo%2C+B">Bakary Diallo</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Beguir%2C+K">Karim Beguir</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Lopez-Carranza%2C+N">Nicolas Lopez-Carranza</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Bent%2C+O">Oliver Bent</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Quantitative Methods
                        (q-bio.QM)</span>; Artificial Intelligence (cs.AI); Machine Learning (cs.LG)

                </div>
                <p class="mathjax">Protein-protein interactions (PPIs) play a crucial role in numerous
                    biological processes. Developing methods that predict binding affinity changes
                    under substitution mutations is fundamental for modelling and re-engineering
                    biological systems. Deep learning is increasingly recognized as a powerful tool
                    capable of bridging the gap between in-silico predictions and in-vitro
                    observations. With this contribution, we propose eGRAL, a novel SE(3)
                    equivariant graph neural network (eGNN) architecture designed for predicting
                    binding affinity changes from multiple amino acid substitutions in protein
                    complexes. eGRAL leverages residue, atomic and evolutionary scales, thanks to
                    features extracted from protein large language models. To address the limited
                    availability of large-scale affinity assays with structural information, we
                    generate a simulated dataset comprising approximately 500,000 data points. Our
                    model is pre-trained on this dataset, then fine-tuned and tested on
                    experimental data.
                </p>
            </div>
        </dd>
        <dt><a name="item2">[2]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02379"
                    title="Abstract">arXiv:2405.02379</a> [<a href="/pdf/2405.02379" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02379" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Modelling the Stochastic Importation Dynamics and
                    Establishment of Novel Pathogenic Strains using a General Branching Processes Framework
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Curran-Sebastian%2C+J">Jacob
                        Curran-Sebastian</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Andersen%2C+F+M">Frederik Mølkjær Andersen</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Bhatt%2C+S">Samir Bhatt</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>; Probability (math.PR); Quantitative Methods (q-bio.QM)

                </div>
                <p class="mathjax">The importation and subsequent establishment of novel pathogenic strains in a
                    population is subject to a large degree of uncertainty due to the stochastic
                    nature of the disease dynamics. Mathematical models need to take this
                    stochasticity in the early phase of an outbreak in order to adequately capture
                    the uncertainty in disease forecasts. We propose a general branching process
                    model of disease spread that includes host-level heterogeneity, and that can be
                    straightforwardly tailored to capture the salient aspects of a particular
                    disease outbreak. We combine this with a model of case importation that occurs
                    via an independent marked Poisson process. We use this framework to investigate
                    the impact of different control strategies, particularly on the time to
                    establishment of an invading, exogenous strain, using parameters taken from the
                    literature for COVID-19 as an example. We also demonstrate how to combine our
                    model with a deterministic approximation, such that longer term projections can
                    be generated that still incorporate the uncertainty from the early growth phase
                    of the epidemic. Our approach produces meaningful short- and medium-term
                    projections of the course of a disease outbreak when model parameters are still
                    uncertain and when stochasticity still has a large effect on the population
                    dynamics.
                </p>
            </div>
        </dd>
        <dt><a name="item3">[3]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02381"
                    title="Abstract">arXiv:2405.02381</a> [<a href="/pdf/2405.02381" title="Download PDF">pdf</a>, <a
                    href="/ps/2405.02381" title="Download PostScript">ps</a>, <a href="/format/2405.02381"
                    title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Interactions between eagles and semidomestic reindeer:
                    lessons learned from field surveys and deterrents
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=van+der+Meiden%2C+A+J">Aemilius Johannes van der
                        Meiden</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Lopez-Peinado%2C+A">Andres Lopez-Peinado</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Sunesson%2C+P">Peter Sunesson</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Emilsson%2C+C">Christian Emilsson</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Singh%2C+N+J">Navinder J Singh</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 17 pages, 4 Figures, 1 Table
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>

                </div>
                <p class="mathjax">Predation by eagles on semi-domesticated reindeer (Rangifer tarandus) is an
                    emerging human wildlife conflict in Fennoscandia. Both the Golden (Aquila
                    chrysaetos) and the White-tailed eagle (Haliaeetus albicilla) are believed by
                    herders to predate on reindeer, however, there is a considerable knowledge gap
                    regarding extent of predation and scavenging by each species, and their
                    distribution and behaviour within the reindeer herding areas. Lethal and
                    non-lethal methods have been suggested to reduce this conflict with eagles. We
                    developed this project to fill the existing knowledge gaps by investigating the
                    patterns of eagle abundance before, during, and after reindeer calving in a
                    herding district in northern Sweden, and testing the effect of two potential
                    deterrents (air ventilators and rotating prisms) in diverting eagles away from
                    reindeer calving areas. During the single study period, we made 12, 47, and 17
                    eagle observations before, during, and after calving respectively. Out of these
                    observations, 34 were of Golden eagles, 33 of White-tailed eagles, and for 9
                    observations the species could not be confirmed. Eagle abundance increased
                    during calving and decreased again after calving ended. No attacks by eagles on
                    calves were observed. The odds of observing eagles were significantly higher in
                    the control area compared to areas with deterrents. More sub-adults were
                    observed during calving, and both species were present in the area. The extent
                    of predation was difficult to infer using direct observations and deterrents
                    seem to show promise in diverting eagles away from calving grounds. These
                    studies should be replicated to get a general picture of the issue and testing
                    the efficiency of deterrents in diverting eagles away from reindeer across
                    reindeer herding districts.
                </p>
            </div>
        </dd>
        <dt><a name="item4">[4]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02524"
                    title="Abstract">arXiv:2405.02524</a> [<a href="/pdf/2405.02524" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02524" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Simulation-based Inference of Developmental EEG Maturation
                    with the Spectral Graph Model
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Bernardo%2C+D">Danilo Bernardo</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Xie%2C+X">Xihe Xie</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Verma%2C+P">Parul Verma</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Kim%2C+J">Jonathan Kim</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Liu%2C+V">Virginia Liu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Wu%2C+Y">Ye Wu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Yap%2C+P">Pew-Thian Yap</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Nagarajan%2C+S">Srikantan Nagarajan</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Raj%2C+A">Ashish Raj</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 58 pages, 6 figures, 18 supplementary figures
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>

                </div>
                <p class="mathjax">The spectral content of macroscopic neural activity evolves throughout
                    development, yet how this maturation relates to underlying brain network
                    formation and dynamics remains unknown. To gain mechanistic insights into this
                    process, we evaluate developmental EEG spectral changes via Bayesian model
                    inversion of the spectral graph model (SGM), a parsimonious model of
                    whole-brain spatiospectral activity derived from linearized neural field models
                    coupled by the structural connectome. Simulation-based inference was used to
                    estimate age-varying SGM parameter posterior distributions from EEG spectra
                    spanning the developmental period. We found this model-fitting approach
                    accurately captures the developmental maturation of EEG spectra via a
                    neurobiologically consistent progression of key neural parameters: long-range
                    coupling, axonal conductance speed, and excitatory:inhibitory balance. These
                    results suggest that spectral maturation of brain activity observed during
                    normal development is supported by functional adaptations, specifically
                    age-dependent tuning of localized neural dynamics and their long-range coupling
                    within the macroscopic, structural network.
                </p>
            </div>
        </dd>
        <dt><a name="item5">[5]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02593"
                    title="Abstract">arXiv:2405.02593</a> [<a href="/pdf/2405.02593" title="Download PDF">pdf</a>, <a
                    href="/ps/2405.02593" title="Download PostScript">ps</a>, <a href="/format/2405.02593"
                    title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> An Interdisciplinary Perspective of the Built-Environment
                    Microbiome
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=McAlister%2C+J+S">John S. McAlister</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Blum%2C+M+J">Michael J. Blum</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Bromberg%2C+Y">Yana Bromberg</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Fefferman%2C+N+H">Nina H. Fefferman</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=He%2C+Q">Qiang He</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Lofgren%2C+E">Eric Lofgren</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Miller%2C+D+L">Debra L. Miller</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Schreiner%2C+C">Courtney Schreiner</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Candan%2C+K+S">K. Selcuk Candan</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Szabo-Rogers%2C+H">Heather Szabo-Rogers</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Reed%2C+J+M">J. Michael Reed</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 23 pages
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>

                </div>
                <p class="mathjax">The built environment provides an excellent setting for interdisciplinary
                    research on the dynamics of microbial communities. The system is simplified
                    compared to many natural settings, and to some extent the entire environment
                    can be manipulated, from architectural design, to materials use, air flow,
                    human traffic, and capacity to disrupt microbial communities through cleaning.
                    Here we provide an overview of the ecology of the microbiome in the built
                    environment. We address niche space and refugia, population and community
                    (metagenomic) dynamics, spatial ecology within a building, including the major
                    microbial transmission mechanisms, as well as evolution. We also address the
                    landscape ecology connecting microbiomes between physically separated
                    buildings. At each stage we pay particular attention to the actual and
                    potential interface between disciplines, such as ecology, epidemiology,
                    materials science, and human social behavior. We end by identifying some
                    opportunities for future interdisciplinary research on the microbiome of the
                    built environment.
                </p>
            </div>
        </dd>
        <dt><a name="item6">[6]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02674"
                    title="Abstract">arXiv:2405.02674</a> [<a href="/pdf/2405.02674" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02674" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Ambush strategy enhances organisms' performance in
                    rock-paper-scissors games
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Barbalho%2C+R">R. Barbalho</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Rodrigues%2C+S">S. Rodrigues</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Tenorio%2C+M">M. Tenorio</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Menezes%2C+J">J. Menezes</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 8 pages, 5 figures
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>; Adaptation and Self-Organizing Systems (nlin.AO); Pattern Formation and
                    Solitons (nlin.PS); Biological Physics (physics.bio-ph); Quantitative Methods (q-bio.QM)

                </div>
                <p class="mathjax">We study a five-species cyclic system wherein individuals of one species
                    strategically adapt their movements to enhance their performance in the spatial
                    rock-paper-scissors game. Environmental cues enable the awareness of the
                    presence of organisms targeted for elimination in the cyclic game. If the local
                    density of target organisms is sufficiently high, individuals move towards
                    concentrated areas for direct attack; otherwise, they employ an ambush tactic,
                    maximising the chances of success by targeting regions likely to be dominated
                    by opponents. Running stochastic simulations, we discover that the ambush
                    strategy enhances the likelihood of individual success compared to direct
                    attacks alone, leading to uneven spatial patterns characterised by spiral
                    waves. We compute the autocorrelation function and measure how the ambush
                    tactic unbalances the organisms' spatial organisation by calculating the
                    characteristic length scale of typical spatial domains of each species. We
                    demonstrate that the threshold for local species density influences the ambush
                    strategy's effectiveness, while the neighbourhood perception range
                    significantly impacts decision-making accuracy. The outcomes show that
                    long-range perception improves performance by over 60\%, although there is
                    potential interference in decision-making under high attack triggers.
                    Understanding how organisms' adaptation to their environment enhances their
                    performance may be helpful not only for ecologists but also for data scientists
                    aiming to improve artificial intelligence systems.
                </p>
            </div>
        </dd>
        <dt><a name="item7">[7]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02767"
                    title="Abstract">arXiv:2405.02767</a> [<a href="/pdf/2405.02767" title="Download PDF">pdf</a>, <a
                    href="/ps/2405.02767" title="Download PostScript">ps</a>, <a href="/format/2405.02767"
                    title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> A Decade in a Systematic Review: The Evolution and Impact of
                    Cell Painting
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Seal%2C+S">Srijit Seal</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Trapotsi%2C+M">Maria-Anna Trapotsi</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Spjuth%2C+O">Ola Spjuth</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Singh%2C+S">Shantanu Singh</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Carreras-Puigvert%2C+J">Jordi
                        Carreras-Puigvert</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Greene%2C+N">Nigel Greene</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Bender%2C+A">Andreas Bender</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Carpenter%2C+A+E">Anne E. Carpenter</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Supplementary Table/Code here: <a
                        href="https://github.com/srijitseal/CellPainting_SystematicReview">this https URL</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Subcellular Processes
                        (q-bio.SC)</span>; Cell Behavior (q-bio.CB)

                </div>
                <p class="mathjax">High-content image-based assays have fueled significant discoveries in the
                    life sciences in the past decade (2013-2023), including novel insights into
                    disease etiology, mechanism of action, new therapeutics, and toxicology
                    predictions. Here, we systematically review the substantial methodological
                    advancements and applications of Cell Painting. Advancements include
                    improvements in the Cell Painting protocol, assay adaptations for different
                    types of perturbations and applications, and improved methodologies for feature
                    extraction, quality control, and batch effect correction. Moreover, machine
                    learning methods recently surpassed classical approaches in their ability to
                    extract biologically useful information from Cell Painting images. Cell
                    Painting data have been used alone or in combination with other -omics data to
                    decipher the mechanism of action of a compound, its toxicity profile, and many
                    other biological effects. Overall, key methodological advances have expanded
                    the ability of Cell Painting to capture cellular responses to various
                    perturbations. Future advances will likely lie in advancing computational and
                    experimental techniques, developing new publicly available datasets, and
                    integrating them with other high-content data types.
                </p>
            </div>
        </dd>
        <dt><a name="item8">[8]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02786"
                    title="Abstract">arXiv:2405.02786</a> [<a href="/pdf/2405.02786" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02786" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> A causal inference approach of monosynapses from spike trains
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Saccomano%2C+Z">Zach Saccomano</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Mckenzie%2C+S">Sam Mckenzie</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Rotstein%2C+H">Horacio Rotstein</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Amarasingham%2C+A">Asohan Amarasingham</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>

                </div>
                <p class="mathjax">Neuroscientists have worked on the problem of estimating synaptic properties,
                    such as connectivity and strength, from simultaneously recorded spike trains
                    since the 1960s. Recent years have seen renewed interest in the problem,
                    coinciding with rapid advances in the technology of high-density neural
                    recordings and optogenetics, which can be used to calibrate causal hypotheses
                    about functional connectivity. Here, a rigorous causal inference framework for
                    pairwise excitatory and inhibitory monosynaptic effects between spike trains is
                    developed. Causal interactions are identified by separating spike interactions
                    in pairwise spike trains by their timescales. Fast algorithms for computing
                    accurate estimates of associated quantities are also developed. Through the
                    lens of this framework, the link between biophysical parameters and statistical
                    definitions of causality between spike trains is examined across a spectrum of
                    dynamical systems simulations. In an idealized setting, we demonstrate a
                    correspondence between the synaptic causal metric developed here and the
                    probabilities of causation developed by Tian and Pearl. Since the probabilities
                    of causation are derived under distinct assumptions and include data from
                    experimental randomization, this opens up the possibility of testing the
                    synaptic inference framework's assumptions with juxtacellular or optogenetic
                    stimulation. We simulate such an experiment with a biophysically detailed
                    channelrhodopsin model and show that randomization is not achieved; strong
                    confounding persists even with strong stimulations. A principal goal is to ask
                    how carefully articulated causal assumptions might better inform the design of
                    neural stimulation experiments and, in turn, support experimental tests of
                    those assumptions.
                </p>
            </div>
        </dd>
        <dt><a name="item9">[9]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02820"
                    title="Abstract">arXiv:2405.02820</a> [<a href="/pdf/2405.02820" title="Download PDF">pdf</a>, <a
                    href="/format/2405.02820" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Coat stiffening explains the consensus pathway of
                    clathrin-mediated endocytosis
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Frey%2C+F">Felix Frey</a> (IST Austria),
                    <a href="/search/q-bio?searchtype=author&amp;query=Schwarz%2C+U+S">Ulrich S. Schwarz</a> (Heidelberg
                    University)
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> revtex, 12 pages, 5 figures in PDF-format
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Subcellular Processes
                        (q-bio.SC)</span>; Soft Condensed Matter (cond-mat.soft)

                </div>
                <p class="mathjax">Clathrin-mediated endocytosis is the main pathway used by eukaryotic cells to
                    take up extracellular material, but the dominant physical mechanisms driving
                    this process are still elusive. Recently several high-resolution imaging
                    techniques have been used on different cell lines to measure the geometrical
                    properties of clathrin-coated pits over their whole lifetime. Here we first
                    show that all datasets follow the same consensus pathway, which is well
                    described by the recently introduced cooperative curvature model, which
                    predicts a flat-to curved transition at finite area, followed by linear growth
                    and subsequent saturation of curvature. We then apply an energetic model for
                    the composite of plasma membrane and clathrin coat to the consensus pathway to
                    show that the dominant mechanism for invagination is coat stiffening, which
                    results from cooperative interactions between the different clathrin molecules
                    and progressively drives the system towards its intrinsic curvature. Our theory
                    predicts that two length scales determine the time course of invagination,
                    namely the patch size at which the flat-to-curved transition occurs and the
                    final pit radius.
                </p>
            </div>
        </dd>
        <dt><a name="item10">[10]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02853"
                    title="Abstract">arXiv:2405.02853</a> [<a href="/pdf/2405.02853" title="Download PDF">pdf</a>, <a
                    href="/ps/2405.02853" title="Download PostScript">ps</a>, <a href="/format/2405.02853"
                    title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Development and validation of a short form of the medication
                    literacy scale for Chinese College Students
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Zhenzhen%2C+C">Chen Zhenzhen</a> (1,2),
                    <a href="/search/q-bio?searchtype=author&amp;query=Jiabao%2C+R">Ren Jiabao</a> (1,2),
                    <a href="/search/q-bio?searchtype=author&amp;query=Tingyu%2C+D">Duan Tingyu</a> (3),
                    <a href="/search/q-bio?searchtype=author&amp;query=Ke%2C+C">Chen Ke</a> (4),
                    <a href="/search/q-bio?searchtype=author&amp;query=Ruyi%2C+H">Hou Ruyi</a> (5),
                    <a href="/search/q-bio?searchtype=author&amp;query=Yimiao%2C+L">Li Yimiao</a> (5),
                    <a href="/search/q-bio?searchtype=author&amp;query=Leixiao%2C+Z">Zeng Leixiao</a> (5),
                    <a href="/search/q-bio?searchtype=author&amp;query=Xiaoxuan%2C+M">Meng Xiaoxuan</a> (6),
                    <a href="/search/q-bio?searchtype=author&amp;query=Yibo%2C+W">Wu Yibo</a> (7),
                    <a href="/search/q-bio?searchtype=author&amp;query=Yu%2C+L">Liu Yu</a> (2), ((1) College of Science,
                    Minzu University of China, Beijing, China, (2) School of Nursing, China Medical University,
                    Shenyang, Liaoning Province, China, (3) Hebei Institute of Communications, Hebei, China, (4)
                    Department of Social Science and Humanities, Harbin Medical University, Harbin, Heilongjiang
                    Province, China, (5) School of Journalism and Communication, Renmin University of China, Beijing,
                    China, (6) Tianjin Medical University, Tianjin, China, (7) School of Public Health, Peking
                    University, Beijing, China)
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 25 pages, 3 figures,3 tables
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Other Quantitative Biology
                        (q-bio.OT)</span>

                </div>
                <p class="mathjax">Medication literacy is integral to health literacy, pivotal for medication
                    safety and adherence. It denotes an individual's capacity to discern,
                    comprehend, and convey medication-related information. Existing scales,
                    however, are time-consuming and predominantly cater to patients and community
                    dwellers, necessitating a more succinct instrument. This study presents the
                    development of a brief Medication Literacy Scale (MLS-14) utilizing classical
                    test theory (CTT) and item response theory (IRT), targeting a college student
                    demographic. The MLS-14's abbreviated version, a 6-item scale (MLS-SF), was
                    distilled through CTT and IRT methodologies, engaging 2431 Chinese college
                    students to scrutinize its psychometric properties. The MLS-SF demonstrated a
                    Cronbach's {\alpha} of 0.765, with three extracted factors via exploratory
                    factor analysis, accounting for 66% of the cumulative variance. All items
                    exhibited factor loadings above 0.5. The scale's three-factor structure was
                    substantiated through confirmatory factor analysis with satisfactory fit
                    indices (chi2/df=5.11, RMSEA=0.063, GFI=0.990, AGFI=0.966, NFI=0.984,
                    IFI=0.987, CFI=0.987). IRT modeling confirmed reasonable discrimination and
                    location parameters for all items, free of differential item functioning (DIF)
                    by gender. Except for items 4 and 10, the remaining items were informative at
                    medium theta levels, indicating their utility in assessing medication literacy
                    efficiently. The developed 6-item Medication Literacy Short Form (MLS-SF)
                    proves to be a reliable and valid instrument for the expedited evaluation of
                    college students' medication literacy, offering a valuable addition to the
                    arsenal of health literacy assessment tools.
                </p>
            </div>
        </dd>
        <dt><a name="item11">[11]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.03148"
                    title="Abstract">arXiv:2405.03148</a> [<a href="/pdf/2405.03148" title="Download PDF">pdf</a>, <a
                    href="/format/2405.03148" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Counting Subnetworks Under Gene Duplication in Genetic
                    Regulatory Networks
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Scruse%2C+A">Ashley Scruse</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Arnold%2C+J">Jonathan Arnold</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Robinson%2C+R">Robert Robinson</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Molecular Networks
                        (q-bio.MN)</span>; Combinatorics (math.CO)

                </div>
                <p class="mathjax">Gene duplication is a fundamental evolutionary mechanism that contributes to
                    biological complexity and diversity (Fortna et al., 2004). Traditionally,
                    research has focused on the duplication of gene sequences (Zhang, 1914).
                    However, evidence suggests that the duplication of regulatory elements may also
                    play a significant role in the evolution of genomic functions (Teichmann and
                    Babu, 2004; Hallin and Landry, 2019). In this work, the evolution of regulatory
                    relationships belonging to gene-specific-substructures in a GRN are modeled. In
                    the model, a network grows from an initial configuration by repeatedly choosing
                    a random gene to duplicate. The likelihood that the regulatory relationships
                    associated with the selected gene are retained through duplication is
                    determined by a vector of probabilities. Occurrences of gene-family-specific
                    substructures are counted under the gene duplication model. In this thesis,
                    gene-family-specific substructures are referred to as subnetwork motifs. These
                    subnetwork motifs are motivated by network motifs which are patterns of
                    interconnections that recur more often in a specialized network than in a
                    random network (Milo et al., 2002). Subnetwork motifs differ from network
                    motifs in the way that subnetwork motifs are instances of gene-family-specific
                    substructures while network motifs are isomorphic substructures. These
                    subnetwork motifs are counted under Full and Partial Duplication, which differ
                    in the way in which regulation relationships are inherited. Full duplication
                    occurs when all regulatory links are inherited at each duplication step, and
                    Partial Duplication occurs when regulation inheritance varies at each
                    duplication step. Moments for the number of occurrences of subnetwork motifs
                    are determined in each model. The results presented offer a method for
                    discovering subnetwork motifs that are significant in a GRN under gene
                    duplication.
                </p>
            </div>
        </dd>
        <dt><a name="item12">[12]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.03346"
                    title="Abstract">arXiv:2405.03346</a> [<a href="/pdf/2405.03346" title="Download PDF">pdf</a>, <a
                    href="/format/2405.03346" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Population dynamics and games of variable size
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Hansen%2C+M">Matheus Hansen</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Chalub%2C+F+A+C+C">Fabio A. C. C. Chalub</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 24 pages, 4 figures
                </div>
                <div class="list-journal-ref">
                    <span class="descriptor">Journal-ref:</span> Matheus Hansen, Fabio A. C. C. Chalub. Population
                    dynamics and
                    games of variable size. Journal of Theoretical Biology 589, 111842, 2024
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>

                </div>
                <p class="mathjax">This work introduces the concept of Variable Size Game Theory (VSGT), in
                    which the number of players in a game is a strategic decision made by the
                    players themselves. We start by discussing the main examples in game theory:
                    dominance, coexistence, and coordination. We show that the same set of pay-offs
                    can result in coordination-like or coexistence-like games depending on the
                    strategic decision of each player type. We also solve an inverse problem to
                    find a <span class="MathJax_Preview" style="display: none;"></span><span class="MathJax"
                        id="MathJax-Element-1-Frame" tabindex="0" style="">
                        <nobr><span class="math" id="MathJax-Span-1"
                                style="width: 0.639em; display: inline-block;"><span
                                    style="display: inline-block; position: relative; width: 0.524em; height: 0px; font-size: 120%;"><span
                                        style="position: absolute; clip: rect(1.045em, 1000.52em, 2.086em, -999.997em); top: -1.907em; left: 0em;"><span
                                            class="mrow" id="MathJax-Span-2"><span class="mi" id="MathJax-Span-3"
                                                style="font-family: MathJax_Math-italic;">d<span
                                                    style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span
                                            style="display: inline-block; width: 0px; height: 1.913em;"></span></span></span><span
                                    style="display: inline-block; overflow: hidden; vertical-align: -0.066em; border-left: 0px solid; width: 0px; height: 0.976em;"></span></span>
                        </nobr>
                    </span>
                    <script type="math/tex" id="MathJax-Element-1">d</script>-player game that reproduces the same
                    fixation pattern of the VSGT.
                    In the sequel, we consider a game involving prosocial and antisocial players,
                    i.e., individuals who tend to play with large groups and small groups,
                    respectively. In this game, a certain task should be performed, that will
                    benefit one of the participants at the expense of the other players. We show
                    that individuals able to gather large groups to perform the task may prevail,
                    even if this task is costly, providing a possible scenario for the evolution of
                    eusociality. The next example shows that different strategies regarding game
                    size may lead to spontaneous separation of different types, a possible scenario
                    for speciation without physical separation (sympatric speciation). In the last
                    example, we generalize to three types of populations from the previous analysis
                    and study compartmental epidemic models: in particular, we recast the SIRS
                    model into the VSGT framework: Susceptibles play 2-player games, while
                    Infectious and Removed play a 1-player game. The SIRS epidemic model is then
                    obtained as the replicator equation of the VSGT. We finish with possible
                    applications of VSGT to be addressed in the future.
                </p>
            </div>
        </dd>
        <dt><a name="item13">[13]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.03370"
                    title="Abstract">arXiv:2405.03370</a> [<a href="/pdf/2405.03370" title="Download PDF">pdf</a>, <a
                    href="/format/2405.03370" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> AntiFold: Improved antibody structure-based design using
                    inverse folding
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=H%C3%B8ie%2C+M+H">Magnus Haraldson Høie</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Hummer%2C+A">Alissa Hummer</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Olsen%2C+T+H">Tobias H. Olsen</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Aguilar-Sanjuan%2C+B">Broncio
                        Aguilar-Sanjuan</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Nielsen%2C+M">Morten Nielsen</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Deane%2C+C+M">Charlotte M. Deane</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Biomolecules
                        (q-bio.BM)</span>; Quantitative Methods (q-bio.QM)

                </div>
                <p class="mathjax">The design and optimization of antibodies requires an intricate balance
                    across multiple properties. Protein inverse folding models, capable of
                    generating diverse sequences folding into the same structure, are promising
                    tools for maintaining structural integrity during antibody design. Here, we
                    present AntiFold, an antibody-specific inverse folding model, fine-tuned from
                    ESM-IF1 on solved and predicted antibody structures. AntiFold outperforms
                    existing inverse folding tools on sequence recovery across
                    complementarity-determining regions, with designed sequences showing high
                    structural similarity to their solved counterpart. It additionally achieves
                    stronger correlations when predicting antibody-antigen binding affinity in a
                    zero-shot manner, while performance is augmented further when including antigen
                    information. AntiFold assigns low probabilities to mutations that disrupt
                    antigen binding, synergizing with protein language model residue probabilities,
                    and demonstrates promise for guiding antibody optimization while retaining
                    structure-related properties. AntiFold is freely available under the BSD
                    3-Clause as a web server at https://opig.stats.ox.ac.uk/webapps/antifold/ and
                    and pip installable package at https://github.com/oxpig/AntiFold
                </p>
            </div>
        </dd>
        <dt><a name="item14">[14]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.03601"
                    title="Abstract">arXiv:2405.03601</a> [<a href="/pdf/2405.03601" title="Download PDF">pdf</a>, <a
                    href="/format/2405.03601" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Firing rate model for brain rhythms controlled by astrocytes
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Stasenko%2C+S+V">Sergey V. Stasenko</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Olenin%2C+S+M">Sergey M. Olenin</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Grines%2C+E+A">Eugene A. Grines</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Levanova%2C+T+A">Tatiana A. Levanova</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 12 pages, 4 figures
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>

                </div>
                <p class="mathjax">We propose a new mean-field model of brain rhythms governed by astrocytes.
                    This theoretical framework describes how astrocytes can regulate neuronal
                    activity and contribute to the generation of brain rhythms. The model describes
                    at the population level the interactions between two large groups of excitatory
                    and inhibitory neurons. The excitatory population is governed by astrocytes via
                    a so-called tripartite synapse. This approach allows us to describe how the
                    interactions between different groups of neurons and astrocytes can give rise
                    to various patterns of synchronized activity and transitions between them.
                    Using methods of nonlinear analysis we show that astrocytic modulation can lead
                    to a change in the period and amplitude of oscillations in the populations of
                    neurons.
                </p>
            </div>
        </dd>
        <dt><a name="item15">[15]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.03602"
                    title="Abstract">arXiv:2405.03602</a> [<a href="/pdf/2405.03602" title="Download PDF">pdf</a>, <a
                    href="/format/2405.03602" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> One nose but two nostrils: Learn to align with sparse
                    connections between two olfactory cortices
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Liu%2C+B">Bo Liu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Qin%2C+S">Shanshan Qin</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Murthy%2C+V">Venkatesh Murthy</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Tu%2C+Y">Yuhai Tu</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>; Biological Physics (physics.bio-ph)

                </div>
                <p class="mathjax">The integration of neural representations in the two hemispheres is an
                    important problem in neuroscience. Recent experiments revealed that odor
                    responses in cortical neurons driven by separate stimulation of the two
                    nostrils are highly correlated. This bilateral alignment points to structured
                    inter-hemispheric connections, but detailed mechanism remains unclear. Here, we
                    hypothesized that continuous exposure to environmental odors shapes these
                    projections and modeled it as online learning with local Hebbian rule. We found
                    that Hebbian learning with sparse connections achieves bilateral alignment,
                    exhibiting a linear trade-off between speed and accuracy. We identified an
                    inverse scaling relationship between the number of cortical neurons and the
                    inter-hemispheric projection density required for desired alignment accuracy,
                    i.e., more cortical neurons allow sparser inter-hemispheric projections. We
                    next compared the alignment performance of local Hebbian rule and the global
                    stochastic-gradient-descent (SGD) learning for artificial neural networks. We
                    found that although SGD leads to the same alignment accuracy with modestly
                    sparser connectivity, the same inverse scaling relation holds. We showed that
                    their similar performance originates from the fact that the update vectors of
                    the two learning rules align significantly throughout the learning process.
                    This insight may inspire efficient sparse local learning algorithms for more
                    complex problems.
                </p>
            </div>
        </dd>
    </dl>
    <h3>Cross-lists for Tue, 7 May 24</h3>
    <dl>
        <dt><a name="item16">[16]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02354"
                    title="Abstract">arXiv:2405.02354</a> (cross-list from cs.LG) [<a href="/pdf/2405.02354"
                    title="Download PDF">pdf</a>, <a href="/ps/2405.02354" title="Download PostScript">ps</a>, <a
                    href="/format/2405.02354" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Heterogeneous network and graph attention auto-encoder for
                    LncRNA-disease association prediction
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Liu%2C+J">Jin-Xing Liu</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Xi%2C+W">Wen-Yu Xi</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Dai%2C+L">Ling-Yun Dai</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Zheng%2C+C">Chun-Hou Zheng</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Gao%2C+Y">Ying-Lian Gao</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 10 pages, 8 figures
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning
                        (cs.LG)</span>; Artificial Intelligence (cs.AI); Quantitative Methods (q-bio.QM)

                </div>
                <p class="mathjax">The emerging research shows that lncRNAs are associated with a series of
                    complex human diseases. However, most of the existing methods have limitations
                    in identifying nonlinear lncRNA-disease associations (LDAs), and it remains a
                    huge challenge to predict new LDAs. Therefore, the accurate identification of
                    LDAs is very important for the warning and treatment of diseases. In this work,
                    multiple sources of biomedical data are fully utilized to construct
                    characteristics of lncRNAs and diseases, and linear and nonlinear
                    characteristics are effectively integrated. Furthermore, a novel deep learning
                    model based on graph attention automatic encoder is proposed, called HGATELDA.
                    To begin with, the linear characteristics of lncRNAs and diseases are created
                    by the miRNA-lncRNA interaction matrix and miRNA-disease interaction matrix.
                    Following this, the nonlinear features of diseases and lncRNAs are extracted
                    using a graph attention auto-encoder, which largely retains the critical
                    information and effectively aggregates the neighborhood information of nodes.
                    In the end, LDAs can be predicted by fusing the linear and nonlinear
                    characteristics of diseases and lncRNA. The HGATELDA model achieves an
                    impressive AUC value of 0.9692 when evaluated using a 5-fold cross-validation
                    indicating its superior performance in comparison to several recent prediction
                    models. Meanwhile, the effectiveness of HGATELDA in identifying novel LDAs is
                    further demonstrated by case studies. the HGATELDA model appears to be a viable
                    computational model for predicting LDAs.
                </p>
            </div>
        </dd>
        <dt><a name="item17">[17]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02449"
                    title="Abstract">arXiv:2405.02449</a> (cross-list from stat.ML) [<a href="/pdf/2405.02449"
                    title="Download PDF">pdf</a>, <a href="/format/2405.02449" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Quality-Weighted Vendi Scores And Their Application To
                    Diverse Experimental Design
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/stat?searchtype=author&amp;query=Nguyen%2C+Q">Quan Nguyen</a>,
                    <a href="/search/stat?searchtype=author&amp;query=Dieng%2C+A+B">Adji Bousso Dieng</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Published in International Conference on Machine Learning,
                    ICML 2024. Code can be found in the Vertaix GitHub: <a
                        href="https://github.com/vertaix/Quality-Weighted-Vendi-Score.">this https URL</a> Paper
                    dedicated to Kwame Nkrumah
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning
                        (stat.ML)</span>; Materials Science (cond-mat.mtrl-sci); Machine Learning (cs.LG); Biomolecules
                    (q-bio.BM)

                </div>
                <p class="mathjax">Experimental design techniques such as active search and Bayesian
                    optimization are widely used in the natural sciences for data collection and
                    discovery. However, existing techniques tend to favor exploitation over
                    exploration of the search space, which causes them to get stuck in local
                    optima. This ``collapse" problem prevents experimental design algorithms from
                    yielding diverse high-quality data. In this paper, we extend the Vendi scores
                    -- a family of interpretable similarity-based diversity metrics -- to account
                    for quality. We then leverage these quality-weighted Vendi scores to tackle
                    experimental design problems across various applications, including drug
                    discovery, materials discovery, and reinforcement learning. We found that
                    quality-weighted Vendi scores allow us to construct policies for experimental
                    design that flexibly balance quality and diversity, and ultimately assemble
                    rich and diverse sets of high-performing data points. Our algorithms led to a
                    70%-170% increase in the number of effective discoveries compared to baselines.
                </p>
            </div>
        </dd>
        <dt><a name="item18">[18]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02534"
                    title="Abstract">arXiv:2405.02534</a> (cross-list from cs.LG) [<a href="/pdf/2405.02534"
                    title="Download PDF">pdf</a>, <a href="/format/2405.02534" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> A Multi-Domain Multi-Task Approach for Feature Selection from
                    Bulk RNA Datasets
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Salta%2C+K">Karim Salta</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Ghosh%2C+T">Tomojit Ghosh</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Kirby%2C+M">Michael Kirby</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning
                        (cs.LG)</span>; Genomics (q-bio.GN)

                </div>
                <p class="mathjax">In this paper a multi-domain multi-task algorithm for feature selection in
                    bulk RNAseq data is proposed. Two datasets are investigated arising from mouse
                    host immune response to Salmonella infection. Data is collected from several
                    strains of collaborative cross mice. Samples from the spleen and liver serve as
                    the two domains. Several machine learning experiments are conducted and the
                    small subset of discriminative across domains features have been extracted in
                    each case. The algorithm proves viable and underlines the benefits of across
                    domain feature selection by extracting new subset of discriminative features
                    which couldn't be extracted only by one-domain approach.
                </p>
            </div>
        </dd>
        <dt><a name="item19">[19]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02564"
                    title="Abstract">arXiv:2405.02564</a> (cross-list from cs.CV) [<a href="/pdf/2405.02564"
                    title="Download PDF">pdf</a>, <a href="/ps/2405.02564" title="Download PostScript">ps</a>, <a
                    href="/format/2405.02564" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Leveraging the Human Ventral Visual Stream to Improve Neural
                    Network Robustness
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Shao%2C+Z">Zhenan Shao</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Ma%2C+L">Linjian Ma</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Li%2C+B">Bo Li</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Beck%2C+D+M">Diane M. Beck</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Computer Vision and Pattern
                        Recognition (cs.CV)</span>; Artificial Intelligence (cs.AI); Neurons and Cognition (q-bio.NC)

                </div>
                <p class="mathjax">Human object recognition exhibits remarkable resilience in cluttered and
                    dynamic visual environments. In contrast, despite their unparalleled
                    performance across numerous visual tasks, Deep Neural Networks (DNNs) remain
                    far less robust than humans, showing, for example, a surprising susceptibility
                    to adversarial attacks involving image perturbations that are (almost)
                    imperceptible to humans. Human object recognition likely owes its robustness,
                    in part, to the increasingly resilient representations that emerge along the
                    hierarchy of the ventral visual cortex. Here we show that DNNs, when guided by
                    neural representations from a hierarchical sequence of regions in the human
                    ventral visual stream, display increasing robustness to adversarial attacks.
                    These neural-guided models also exhibit a gradual shift towards more human-like
                    decision-making patterns and develop hierarchically smoother decision surfaces.
                    Importantly, the resulting representational spaces differ in important ways
                    from those produced by conventional smoothing methods, suggesting that such
                    neural-guidance may provide previously unexplored robustness solutions. Our
                    findings support the gradual emergence of human robustness along the ventral
                    visual hierarchy and suggest that the key to DNN robustness may lie in
                    increasing emulation of the human brain.
                </p>
            </div>
        </dd>
        <dt><a name="item20">[20]</a>&nbsp; <span class="list-identifier"><a href="/abs/2405.02845"
                    title="Abstract">arXiv:2405.02845</a> (cross-list from cs.LG) [<a href="/pdf/2405.02845"
                    title="Download PDF">pdf</a>, <a href="/format/2405.02845" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Data-Efficient Molecular Generation with Hierarchical Textual
                    Inversion
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Kim%2C+S">Seojin Kim</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Nam%2C+J">Jaehyun Nam</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Yu%2C+S">Sihyun Yu</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Shin%2C+Y">Younghoon Shin</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Shin%2C+J">Jinwoo Shin</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning
                        (cs.LG)</span>; Molecular Networks (q-bio.MN)

                </div>
                <p class="mathjax">Developing an effective molecular generation framework even with a limited
                    number of molecules is often important for its practical deployment, e.g., drug
                    discovery, since acquiring task-related molecular data requires expensive and
                    time-consuming experimental costs. To tackle this issue, we introduce
                    Hierarchical textual Inversion for Molecular generation (HI-Mol), a novel
                    data-efficient molecular generation method. HI-Mol is inspired by the
                    importance of hierarchical information, e.g., both coarse- and fine-grained
                    features, in understanding the molecule distribution. We propose to use
                    multi-level embeddings to reflect such hierarchical features based on the
                    adoption of the recent textual inversion technique in the visual domain, which
                    achieves data-efficient image generation. Compared to the conventional textual
                    inversion method in the image domain using a single-level token embedding, our
                    multi-level token embeddings allow the model to effectively learn the
                    underlying low-shot molecule distribution. We then generate molecules based on
                    the interpolation of the multi-level token embeddings. Extensive experiments
                    demonstrate the superiority of HI-Mol with notable data-efficiency. For
                    instance, on QM9, HI-Mol outperforms the prior state-of-the-art method with 50x
                    less training data. We also show the effectiveness of molecules generated by
                    HI-Mol in low-shot molecular property prediction.
                </p>
            </div>
        </dd>
    </dl>
    <h3>Replacements for Tue, 7 May 24</h3>
    <dl>
        <dt><a name="item21">[21]</a>&nbsp; <span class="list-identifier"><a href="/abs/2208.06348"
                    title="Abstract">arXiv:2208.06348</a> (replaced) [<a href="/pdf/2208.06348"
                    title="Download PDF">pdf</a>, <a href="/format/2208.06348" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Can Brain Signals Reveal Inner Alignment with Human
                    Languages?
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Han%2C+W">William Han</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Qiu%2C+J">Jielin Qiu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Zhu%2C+J">Jiacheng Zhu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Xu%2C+M">Mengdi Xu</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Weber%2C+D">Douglas Weber</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Li%2C+B">Bo Li</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Zhao%2C+D">Ding Zhao</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> EMNLP 2023 Findings
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>; Artificial Intelligence (cs.AI); Computation and Language (cs.CL); Machine
                    Learning (cs.LG)

                </div>
            </div>
        </dd>
        <dt><a name="item22">[22]</a>&nbsp; <span class="list-identifier"><a href="/abs/2301.01445"
                    title="Abstract">arXiv:2301.01445</a> (replaced) [<a href="/pdf/2301.01445"
                    title="Download PDF">pdf</a>, <a href="/format/2301.01445" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> A compositional account of motifs, mechanisms, and dynamics
                    in biochemical regulatory networks
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Aduddell%2C+R">Rebekah Aduddell</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Fairbanks%2C+J">James Fairbanks</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Kumar%2C+A">Amit Kumar</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Ocal%2C+P+S">Pablo S. Ocal</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Patterson%2C+E">Evan Patterson</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Shapiro%2C+B+T">Brandon T. Shapiro</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Final version published in Compositionality
                </div>
                <div class="list-journal-ref">
                    <span class="descriptor">Journal-ref:</span> Compositionality, Volume 6, Issue 2 (2024)
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Molecular Networks
                        (q-bio.MN)</span>; Category Theory (math.CT)

                </div>
            </div>
        </dd>
        <dt><a name="item23">[23]</a>&nbsp; <span class="list-identifier"><a href="/abs/2305.19367"
                    title="Abstract">arXiv:2305.19367</a> (replaced) [<a href="/pdf/2305.19367"
                    title="Download PDF">pdf</a>, <a href="/format/2305.19367" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Fibration symmetries and cluster synchronization in the
                    Caenorhabditis elegans connectome
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Avila%2C+B">Bryant Avila</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Augusto%2C+P">Pedro Augusto</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Zimmer%2C+M">Manuel Zimmer</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Serafino%2C+M">Matteo Serafino</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Makse%2C+H+A">Hernán A. Makse</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Word count Text: 12054 words. 99 references. 17 figures
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>; Quantitative Methods (q-bio.QM)

                </div>
            </div>
        </dd>
        <dt><a name="item24">[24]</a>&nbsp; <span class="list-identifier"><a href="/abs/2307.16767"
                    title="Abstract">arXiv:2307.16767</a> (replaced) [<a href="/pdf/2307.16767"
                    title="Download PDF">pdf</a>, <a href="/ps/2307.16767" title="Download PostScript">ps</a>, <a
                    href="/format/2307.16767" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Infection-induced Cascading Failures -- Impact and Mitigation
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/physics?searchtype=author&amp;query=Li%2C+B">Bo Li</a>,
                    <a href="/search/physics?searchtype=author&amp;query=Saad%2C+D">David Saad</a>
                </div>
                <div class="list-journal-ref">
                    <span class="descriptor">Journal-ref:</span> Communications Physics 7, 144 (2024)
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Physics and Society
                        (physics.soc-ph)</span>; Disordered Systems and Neural Networks (cond-mat.dis-nn); Optimization
                    and Control (math.OC); Populations and Evolution (q-bio.PE)

                </div>
            </div>
        </dd>
        <dt><a name="item25">[25]</a>&nbsp; <span class="list-identifier"><a href="/abs/2310.12952"
                    title="Abstract">arXiv:2310.12952</a> (replaced) [<a href="/pdf/2310.12952"
                    title="Download PDF">pdf</a>, <a href="/format/2310.12952" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Cousins Of The Vendi Score: A Family Of Similarity-Based
                    Diversity Metrics For Science And Machine Learning
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Pasarkar%2C+A+P">Amey P. Pasarkar</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Dieng%2C+A+B">Adji Bousso Dieng</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Published in the proceedings of Artificial Intelligence
                    and Statistics, AISTATS 2024. This paper is dedicated to Aline Sitoe Diatta. The code can be found
                    on Vertaix's GitHub. Code for evaluating diversity using the Vendi scores can be found at <a
                        href="https://github.com/vertaix/Vendi-Score.">this https URL</a> Code for using the scores
                    within Vendi Sampling can be found at <a href="https://github.com/vertaix/Vendi-Sampling">this https
                        URL</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning
                        (cs.LG)</span>; Chemical Physics (physics.chem-ph); Populations and Evolution (q-bio.PE)

                </div>
            </div>
        </dd>
        <dt><a name="item26">[26]</a>&nbsp; <span class="list-identifier"><a href="/abs/2402.13392"
                    title="Abstract">arXiv:2402.13392</a> (replaced) [<a href="/pdf/2402.13392"
                    title="Download PDF">pdf</a>, <a href="/format/2402.13392" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> An SEIR network epidemic model with manual and digital
                    contact tracing allowing delays
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/physics?searchtype=author&amp;query=Zhang%2C+D">Dongni Zhang</a>,
                    <a href="/search/physics?searchtype=author&amp;query=Britton%2C+T">Tom Britton</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Physics and Society
                        (physics.soc-ph)</span>; Probability (math.PR); Populations and Evolution (q-bio.PE)

                </div>
            </div>
        </dd>
        <dt><a name="item27">[27]</a>&nbsp; <span class="list-identifier"><a href="/abs/2403.03134"
                    title="Abstract">arXiv:2403.03134</a> (replaced) [<a href="/pdf/2403.03134"
                    title="Download PDF">pdf</a>, <a href="/format/2403.03134" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Simplicity in Complexity : Explaining Visual Complexity using
                    Deep Segmentation Models
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Shen%2C+T">Tingke Shen</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Nath%2C+S+S">Surabhi S Nath</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Brielmann%2C+A">Aenne Brielmann</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Dayan%2C+P">Peter Dayan</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Computer Vision and Pattern
                        Recognition (cs.CV)</span>; Artificial Intelligence (cs.AI); Neurons and Cognition (q-bio.NC)

                </div>
            </div>
        </dd>
        <dt><a name="item28">[28]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.05468"
                    title="Abstract">arXiv:2404.05468</a> (replaced) [<a href="/pdf/2404.05468"
                    title="Download PDF">pdf</a>, <a href="/format/2404.05468" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Mind-to-Image: Projecting Visual Mental Imagination of the
                    Brain from fMRI
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Caselles-Dupr%C3%A9%2C+H">Hugo
                        Caselles-Dupré</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Mellerio%2C+C">Charles Mellerio</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=H%C3%A9rent%2C+P">Paul Hérent</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Lopez-Persem%2C+A">Alizée Lopez-Persem</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=B%C3%A9ranger%2C+B">Benoit Béranger</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Soularue%2C+M">Mathieu Soularue</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Fautrel%2C+P">Pierre Fautrel</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Vernier%2C+G">Gauthier Vernier</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Cord%2C+M">Matthieu Cord</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> Pre-print to be updated. Work in progress
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition
                        (q-bio.NC)</span>; Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)

                </div>
            </div>
        </dd>
        <dt><a name="item29">[29]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.05696"
                    title="Abstract">arXiv:2404.05696</a> (replaced) [<a href="/pdf/2404.05696"
                    title="Download PDF">pdf</a>, <a href="/ps/2404.05696" title="Download PostScript">ps</a>, <a
                    href="/format/2404.05696" title="Other formats">other</a>]</span></dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> BOLD v4: A Centralized Bioinformatics Platform for DNA-based
                    Biodiversity Data
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cs?searchtype=author&amp;query=Ratnasingham%2C+S">Sujeevan Ratnasingham</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Wei%2C+C">Catherine Wei</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Chan%2C+D">Dean Chan</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Agda%2C+J">Jireh Agda</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Agda%2C+J">Josh Agda</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Ballesteros-Mejia%2C+L">Liliana
                        Ballesteros-Mejia</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Boutou%2C+H+A">Hamza Ait Boutou</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Bastami%2C+Z+M+E">Zak Mohammad El Bastami</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Ma%2C+E">Eddie Ma</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Manjunath%2C+R">Ramya Manjunath</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Rea%2C+D">Dana Rea</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Ho%2C+C">Chris Ho</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Telfer%2C+A">Angela Telfer</a>,
                    <a href="/search/cs?searchtype=author&amp;query=McKeowan%2C+J">Jaclyn McKeowan</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Rahulan%2C+M">Miduna Rahulan</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Steinke%2C+C">Claudia Steinke</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Dorsheimer%2C+J">Justin Dorsheimer</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Milton%2C+M">Megan Milton</a>,
                    <a href="/search/cs?searchtype=author&amp;query=Hebert%2C+P+D+N">Paul D. N. Hebert</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Databases (cs.DB)</span>;
                    Quantitative Methods (q-bio.QM)

                </div>
            </div>
        </dd>
        <dt><a name="item30">[30]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.17026"
                    title="Abstract">arXiv:2404.17026</a> (replaced) [<a href="/pdf/2404.17026"
                    title="Download PDF">pdf</a>, <a href="/format/2404.17026" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Catalytic Coagulation
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/cond-mat?searchtype=author&amp;query=Krapivsky%2C+P+L">P. L. Krapivsky</a>,
                    <a href="/search/cond-mat?searchtype=author&amp;query=Redner%2C+S">S. Redner</a>
                </div>
                <div class="list-comments mathjax">
                    <span class="descriptor">Comments:</span> 8 pages, 1 figure. Version 2: terminology of the model
                    changed. No other changes
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Statistical Mechanics
                        (cond-mat.stat-mech)</span>; Biological Physics (physics.bio-ph); Chemical Physics
                    (physics.chem-ph); Quantitative Methods (q-bio.QM)

                </div>
            </div>
        </dd>
        <dt><a name="item31">[31]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19041"
                    title="Abstract">arXiv:2404.19041</a> (replaced) [<a href="/pdf/2404.19041"
                    title="Download PDF">pdf</a>, <a href="/format/2404.19041" title="Other formats">other</a>]</span>
        </dt>
        <dd>
            <div class="meta">
                <div class="list-title mathjax">
                    <span class="descriptor">Title:</span> Stochastic dynamics of two-compartment models with regulatory
                    mechanisms for hematopoiesis
                </div>
                <div class="list-authors">
                    <span class="descriptor">Authors:</span>
                    <a href="/search/q-bio?searchtype=author&amp;query=Wang%2C+R">Ren-Yi Wang</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Kimmel%2C+M">Marek Kimmel</a>,
                    <a href="/search/q-bio?searchtype=author&amp;query=Pang%2C+G">Guodong Pang</a>
                </div>
                <div class="list-subjects">
                    <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution
                        (q-bio.PE)</span>

                </div>
            </div>
        </dd>
    </dl>
    <ul>
        <li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
        <li><a href="#item16">Cross-lists</a></li>
        <li><a href="#item21">Replacements</a></li>
    </ul>
    <small>[ total of 31 entries: <b>1-31</b> ]</small><br>
    <small>[ showing up to 2000 entries per page: <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> | <font
            color="#999999">more</font> ]</small><br>
</div>